NEWS

```diff
@@ -158,6 +158,17 @@ CHANGES WITH 247 in spe:
           mounting additional disk images into the file system tree accessible
           to the service.
 
+        * Timer units gained a new FixedRandomDelay= boolean setting. If
+          enabled the random delay configured with RandomizedDelaySec= is
+          hashed from the unit name, system identity, and execution context, so
+          that always the same offset is used for the same unit on the same
```

Comment:
```that always the same offset is used``` -> ```that the same offset is always used```

Reply:
I'd skip those precise details of implementation. The man page uses the same phrasing, and I think it's too much there too. I'll submit a follow-up pr.